uniflow/flow/transform/model.py

```diff
@@ -17,101 +17,9 @@
 MAX_ATTEMPTS = 3
 
 
-class Model:
+class LLMDataProcessorJson(LLMDataProcessor):
```

Comment:
nit: this is a very confusing name....

Reply:
If you really want to have json in the name. it should be `LLMJasonDataPreprocessor`, but this name sounds too long for me.

Reply:
`LLMJsonDataPreprocessor` sounds like the input data is JSON... that's why I put JSON in the last